pkgs/tools/security/yubihsm-connector/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, libusb, buildGoModule, fetchFromGitHub, pkg-config }:
+
+buildGoModule rec {
+  pname = "yubihsm-connector";
+  version = "3.0.2";
+
+  src = fetchFromGitHub {
+    owner = "Yubico";
+    repo = "yubihsm-connector";
+    rev = version;
+    sha256 = "sha256-FQ64tSZN55QpXjMZITzlWOPTKSgnoCpkRngQUQHVavc=";
+  };
+  vendorSha256 = "sha256-kVBzdJk/1LvjdUtLqHAw9ZxDfCo3mBWVMYG/nQXpDrk=";
+
+  patches = [ ./lockfile-fix.patch ];
+
+  preBuild = "go generate";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ libusb ];
+
+  meta = with lib; {
+    homepage = "https://developers.yubico.com/yubihsm-connector/";
```

Comment:
Missing description.

Reply:
There is no upstream description. Should I make up my own?

Reply:
I've added what I found here https://developers.yubico.com/YubiHSM2/Component_Reference/yubihsm-connector/